piptools/utils.py

```diff
@@ -522,3 +531,126 @@ def parse_requirements_from_wheel_metadata(
             markers=parts.markers,
             extras=parts.extras,
         )
+
+
+def callback_config_file_defaults(
+    ctx: click.Context, param: click.Parameter, value: str | None
+) -> str | None:
+    """
+    Returns the path to the config file with defaults being used, or `None` if no such file is
+    found.
```

Comment:
Let's not forget that the docstring title should be in imperative mood, short and simple, referencing to the action being performed, not describing the entire runtime logic. PEP 8 requires such indented blocks not to exceed the limit of 72 chars:
```suggestion
    """Return the config file path.

    ``None`` is returned if no such file is found.
```
(also, we should only use RST in docstrings)